docs/quickstart.rst

```diff
@@ -1,6 +1,68 @@
 Quickstart
 ==========
 
+Use *septentrion* as your migration tool in 2 easy steps:
+  1. Write your migrations.
+  2. Run your migrations.
+
+Write migrations
+----------------
+Your migration folder should be organised as follow:
+ - each version has its own folder;
+ - in a version folder, migrations will be executed in alphabetical order;
+ - `schemas` folder contains state of the schema at a given version.
+
+# I'm not sure if the explanation around the naming convention makes sense in a quick start, probably not?
+# Do we need something more in that section?
+
+File naming convention
+^^^^^^^^^^^^^^^^^^^^^^
+
+We recommend to use the following naming convention for migration files:
+
+.. code-block:: console
+
+    [VERSION]-[MIGRATION]-[ORDER]-[TYPE].sql
+
+ - version : the release version
+
+ - migration : a free name for the migration
+
+ - order : a number used to order the files inside the migration
+
+ - type : ddl or dml, ddl if the file contains schema migration, dml if the file contains data migration.
+
+
+Run migrations
+--------------
+Configure database connection settings
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Ensure that your database connection settings are correctly configured.
+
+ - Either your environment variables `PGHOST`, `PGPORT` and `PGUSER` are properly set.
+ - Or you can set the environment variables `SEPTENTRION_HOST`, `SEPTENTRION_PORT`, `SEPTENTRION_USERNAME`.
+ - If you don't want to use environment variables, you can use the `--host`, `--port` or `--username` options
+   when running the migration.
```

Comment:
I like when there's as little choice as possible in the tutorial part. Also, I think we should avoid assuming the user knows anything as much as possible (for instance how to setup a postgres DB), so there should be a recommanded path and a link for any other pass.

Example: https://procrastinate.readthedocs.io/en/latest/quickstart.html#prerequisites-installation